### PR TITLE
initial style to patch

### PR DESCRIPTION
**Pull request by tarifulfahim**

### CODE REVIEW

**Comment by Arif-Shahriar028** [comment]
why you do this pull requ

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.body{
```
Comment (Arif-Shahriar028) [inline style.css:1]: what do you mean by that

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+    background-color: red;
```
Comment (Arif-Shahriar028) [inline style.css:2]: why red?? green is better

File: style.css (hunk 1 above)
Comment (tarifulfahim) [reply style.css:1]: because it is a danger sign

**Comment by tarifulfahim** [comment]
i don't want to tell you

**Pull request merged**